### DISCUSSION TITLE
新人冒泡

### DISCUSSION THREAD

**Discussion by yzhfat** [share121/inter-knot#5143] [其他]
这是什么？绳网？戳一下

**Comment by qingshuiglaze** [comment]
这是什么？绳网？戳一下！
这是什么？！绳网？！戳一下！！
这是什么？！！绳网？！！戳一下！！！

**Comment by March-777th** [comment]
戳戳你的<img src="https://img-static.mihoyo.com/communityweb/upload/a8b2f0d6c5173547b23b7557d9595391.png" alt="三月七_悄悄话" width="75" >
